### DISCUSSION TITLE
Maintainer Needed

### DISCUSSION THREAD

**Discussion by xloem** [encode/orm#97] [General]
It looks like the project maintainers may have abandoned this project, given it has been a year since the last commit.

@tomchristie @florimondmanca @rzane @thiagovarela @dopry @kcrebound @Lynskylate
is this true, that the project is no longer maintained?

@aminalaee #95 @pykulytsky #92 @GDWR #91 @david-alm #86 @Blue-Hope #76 @AndTheDaysGoBy #73 @asnelling #71
Are any of you available to take on this project and maintain or fork it?

**Comment by lovelydinosaur** [comment]
Hi @xloem - So we can easily add anyone onto the maintainers team, and help folks through what's required to release new versions to PyPI. If anyone wants to be added to @encode/maintainers, just ask.

**Reply by aminalaee** [reply]
@xloem @tomchristie I think I can help with orm and database.

**Reply by lovelydinosaur** [reply]
@aminalaee Okay, let's wing things this ways around. As soon as you've got a pull request that you'd like to merge, that's not being progressed, then ask and will add you to the maintainers team.

**Reply by aminalaee** [reply]
@tomchristie  There's already one #95. To be honest I didn't see any progress with that so I didn't continue. But that's fair, I can create another one.

**Reply by lovelydinosaur** [reply]
Invite sent. Once you've accepted that you'll have the required permissions to merge pull requests etc.
We can also walk you through the release process once we get to that point.

The chat group can also be a useful place to ask about any maintenance questions... https://gitter.im/encode/community

**Reply by blue-hope** [reply]
@tomchristie  I have already one https://github.com/encode/orm/pull/76 too. To be honest, I'm not sure I can say for sure if this feature is really needed for the orm repository, but I'd be excited if I could contribute in this way.

**Reply by lovelydinosaur** [reply]
@Blue-Hope Invited.

**Reply by Mng-dev-ai** [reply]
@tomchristie What do you think about #148 ?
And also I don't know if I can be a maintainer or not but I still have a lot of things I wanna add besides #146 #140 #148 such as values,values_list,only,defer,m2m support and more.

**Comment by roks0n** [comment]
Hi, i see a bugfix (https://github.com/encode/orm/issues/162) that I'd love to get released. I'm sure there are many other useful changes since the last release (29th Nov 2021). I understand I don't have any track record on this repository so asking to become a maintainer might be a bit far fetched.

Is there anything I can do to help get the latest master branch released?
